.gitignore

```diff
@@ -36,5 +36,6 @@ nosetests.xml
 .pydevproject
 
 node_modules
+package-lock.json
```

Comment:
> This file is intended to be committed into source repositories

https://docs.npmjs.com/configuring-npm/package-lock-json.html

Reply:
i see, then i should remove it again i guess, thanks

Reply:
fixed

Reply:
Yes, and once you've done a clean `npm install`, you should also commit this file.

Reply:
should we add this information in the readme?